server/db/index.js

```diff
@@ -55,25 +50,27 @@ function initDb(force = false) {
     Comment.belongsTo(StretchAnswer)
     StretchAnswer.hasMany(Comment)
 
+    // -------------------------------------------------------
+    /* Attendance-related relationships
+
+       Attendance model is a join table that joins students to cohorts.
+       Withdrawl model is a join table that joins students to cohorts.
+    */
+    Attendance.belongsTo(User)
+    User.hasMany(Attendance)
+
+    Attendance.belongsTo(Cohort)
+    Cohort.hasMany(Attendance)
+
+    Withdrawl.belongsTo(User)
```

Comment:
I think it's "withdrawal" - just to prevent confusion later in case someone is trying to make a change and getting an error